File: prototype/musicforecast/src/mongo.js
const mongoose = require('mongoose');
const uri = "mongodb+srv://projectuser:p5comgroup@groupproject411.c2lxw0b.mongodb.net/?retryWrites=true&w=majority";

module.exports = async () => {
    await mongoose.connect(uri, {
        useNewUrlParser: true,
        useUnifiedTopology: true,
    })

    console.log('connecting');

    return mongoose;
}

File: prototype/musicforecast/src/mongo.py
import pymongo
from pymongo import MongoClient
from motor.motor_asyncio import AsyncIOMotorClient

uri = "mongodb+srv://projectuser:p5comgroup@groupproject411.c2lxw0b.mongodb.net/?retryWrites=true&w=majority"
async def database_connect():
    client = AsyncIOMotorClient(uri)
    print ("connecting")
    return client
    